<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setparameter</class>
 <widget class="QDialog" name="setparameter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#wt_1{border:1px solid #666666;border-radius:10px;}</string>
  </property>
  <widget class="QWidget" name="wt_1" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QLabel" name="lb_pagename">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_cancel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>610</y>
      <width>190</width>
      <height>70</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_confirm">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>610</y>
      <width>190</width>
      <height>70</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_line1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>62</y>
      <width>1240</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(160, 160, 160);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>1241</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLabel" name="lb_voice">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_retention">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>80</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_offfire">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_readconf">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>310</y>
        <width>200</width>
        <height>60</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lb_temp">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>200</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_standby">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>80</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_temp">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>190</y>
        <width>150</width>
        <height>60</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lb_dispress">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>370</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_offfire">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>150</width>
        <height>60</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_retention">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>70</y>
        <width>150</width>
        <height>60</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_standby">
      <property name="geometry">
       <rect>
        <x>985</x>
        <y>70</y>
        <width>150</width>
        <height>60</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_voice">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>190</y>
        <width>150</width>
        <height>60</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_recover">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>310</y>
        <width>200</width>
        <height>60</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_reboot">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>310</y>
        <width>200</width>
        <height>60</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <zorder>le_offfire</zorder>
     <zorder>lb_voice</zorder>
     <zorder>lb_retention</zorder>
     <zorder>lb_offfire</zorder>
     <zorder>lb_temp</zorder>
     <zorder>lb_standby</zorder>
     <zorder>le_temp</zorder>
     <zorder>lb_dispress</zorder>
     <zorder>bt_readconf</zorder>
     <zorder>le_retention</zorder>
     <zorder>le_standby</zorder>
     <zorder>le_voice</zorder>
     <zorder>bt_recover</zorder>
     <zorder>bt_reboot</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLabel" name="lb_wifi">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>101</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_wifi">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_4G">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>30</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="wt_pwd" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>1221</width>
        <height>251</height>
       </rect>
      </property>
      <widget class="QLabel" name="lb_serverPort">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>160</y>
         <width>111</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_ip">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>150</y>
         <width>250</width>
         <height>60</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_port">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>150</y>
         <width>250</width>
         <height>60</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
      </widget>
      <widget class="QLabel" name="lb_serverIp">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>101</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_APN">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>40</y>
         <width>250</width>
         <height>60</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
      </widget>
      <widget class="QLabel" name="lb_user">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>50</y>
         <width>101</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lb_APN">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>101</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_user">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>40</y>
         <width>250</width>
         <height>60</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_pwd">
       <property name="geometry">
        <rect>
         <x>950</x>
         <y>40</y>
         <width>250</width>
         <height>60</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
      </widget>
      <widget class="QLabel" name="lb_pwd">
       <property name="geometry">
        <rect>
         <x>840</x>
         <y>50</y>
         <width>101</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <zorder>lb_serverPort</zorder>
      <zorder>le_port</zorder>
      <zorder>lb_serverIp</zorder>
      <zorder>le_ip</zorder>
      <zorder>le_APN</zorder>
      <zorder>lb_user</zorder>
      <zorder>lb_APN</zorder>
      <zorder>le_user</zorder>
      <zorder>le_pwd</zorder>
      <zorder>lb_pwd</zorder>
     </widget>
     <widget class="QLabel" name="lb_4G">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>40</y>
        <width>111</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QWidget" name="wt_wifi" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>109</y>
        <width>1221</width>
        <height>341</height>
       </rect>
      </property>
      <widget class="QLabel" name="lb_now">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>131</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="lb_ssid">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>231</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="lb_wifiIp">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>200</y>
         <width>231</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QListWidget" name="lt_wifi">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>0</y>
         <width>720</width>
         <height>330</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="bt_refresh">
       <property name="geometry">
        <rect>
         <x>1000</x>
         <y>0</y>
         <width>150</width>
         <height>80</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="lb_nowip">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>131</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
     <zorder>wt_wifi</zorder>
     <zorder>wt_pwd</zorder>
     <zorder>lb_wifi</zorder>
     <zorder>bt_wifi</zorder>
     <zorder>bt_4G</zorder>
     <zorder>lb_4G</zorder>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>页</string>
     </attribute>
     <widget class="QLabel" name="lb_version">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_sn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_mode">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_ip">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_mac">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_dismac">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>190</y>
        <width>300</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_disip">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>150</y>
        <width>300</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_disversion">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>300</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_dissn">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>300</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_dismode">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>300</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>bt_cancel</zorder>
   <zorder>bt_confirm</zorder>
   <zorder>lb_pagename</zorder>
   <zorder>lb_line1</zorder>
   <zorder>tabWidget</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
